src/data/mv_polynomial/variables.lean

```diff
@@ -473,6 +473,29 @@ finset.sup_le $ assume n hn,
     { exact le_max_of_le_right (finset.le_sup this) }
   end
 
+lemma total_degree_add_of_total_degree_lt (p q : mv_polynomial σ R)
```

Comment:
Could you please also add the `q + p` version? Maybe call them `total_degree_add_eq_{left,right}`?

Reply:
Done, I also added a symm_diff version of the support lemma.
Yeah I'm not sure on naming? I just checked and the version for univariate polynomials is called `degree_add_eq_left_of_degree_lt` so maybe best to match that?

Reply:
I changed it to match the polynomial version